qgrid/qgridjs/qgrid.js

```diff
@@ -72,7 +72,7 @@ define([
 
     var row_count = 0;
     _.each(this.data_frame, function (cur_row, key, list) {
-      cur_row.id = "row" + row_count;
+      //cur_row.id = "row" + row_count;
```

Comment:
I actually tested this before i submitted the pull request and the slick grid did render.  Looking at the stack overflow discussing the id column though, it appears that the advanced functionality of the data view, which i did not test, requires the id column.

This was my first pull request, thanks for your comments and going easy on me.

-D